$OpenBSD: patch-panels_datetime_datetime_ui,v 1.1.1.1 2011/06/16 10:07:57 ajacoutot Exp $

XXX don't show the NTP switch -- it requires /sbin/service, chkconfig or
update-rc...

--- panels/datetime/datetime.ui.orig	Thu Jun 16 07:29:13 2011
+++ panels/datetime/datetime.ui	Thu Jun 16 07:28:51 2011
@@ -114,7 +114,7 @@
                         <property name="spacing">12</property>
                         <child>
                           <object class="GtkLabel" id="label3">
-                            <property name="visible">True</property>
+                            <property name="visible">False</property>
                             <property name="xalign">0</property>
                             <property name="label" translatable="yes">Network Time</property>
                           </object>
@@ -124,7 +124,7 @@
                         </child>
                         <child>
                           <object class="GtkSwitch" id="network_time_switch">
-                            <property name="visible">True</property>
+                            <property name="visible">False</property>
                             <property name="can_focus">True</property>
                             <property name="receives_default">True</property>
                           </object>
